### PR TITLE
correct ColorPicker/Dropzone/Carousel class name

### LINKED ISSUE #1481: Correct class name in `Carousel`
### Which package has an issue?

@yamada-ui/react

### Package Version

1.3.9

### Description

Corrected the following class names according to BEM

- `ui-carousel__indicators__indicator`
  - fix to `ui-carousel__indicator`
- `ui-carousel__slide__inner`
  - fix to `ui-carousel__slide-inner`
- `ui-carousel__sliders__inner`
  - fix to `ui-carousel__sliders-inner`

Create the changeset with `major`.
Please derive a new branch from the `v2.0` branch and create a pull request for the `v2.0` branch, not the `main` branch, as we plan to release the product in `v2.0.0`.

### Link to Reproduction

_No response_

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

### Do you know how to fix the issue?

None

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### Additional Information

_No response_

### LINKED ISSUE #1482: Correct class name in `Dropzone`
### Which package has an issue?

@yamada-ui/react

### Package Version

1.3.9

### Description

Corrected the following class names according to BEM

- `ui-dropzone__overlay__loading`
  - fix to `ui-dropzone__loading`

Create the changeset with `major`.
Please derive a new branch from the `v2.0` branch and create a pull request for the `v2.0` branch, not the `main` branch, as we plan to release the product in v2.0.0.

### Link to Reproduction

_No response_

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

### Do you know how to fix the issue?

None

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### Additional Information

_No response_

### LINKED ISSUE #1483: Correct class name in `ColorPicker`
### Which package has an issue?

@yamada-ui/react

### Package Version

1.3.9

### Description

Corrected the following class names according to BEM

- `ui-color-picker__eye-dropper__icon`
  - fix to `ui-color-picker__eye-dropper-icon`
- `ui-color-selector__eye-dropper__icon`
  - fix to `ui-color-selector__eye-dropper-icon`
- `ui-color-selector__swatches__label`
  - fix to `ui-color-selector__swatches-label`

Create the changeset with `major`.
Please derive a new branch from the `v2.0` branch and create a pull request for the `v2.0` branch, not the `main` branch, as we plan to release the product in v2.0.0.

### Link to Reproduction

_No response_

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

### Do you know how to fix the issue?

None

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by aojunhao123**
Closes #1481 
Closes #1482 
Closes #1483

### CODE REVIEW

**Review by koralle — DISMISSED** [review]

File: .changeset/sweet-monkeys-drive.md
Comment (illionillion) [inline .changeset/sweet-monkeys-drive.md:1]: @aojunhao123 
Are these really the correct packages that have been changed?

Comment (aojunhao123) [reply .changeset/sweet-monkeys-drive.md:1]: ok，I will fix it.

Comment (aojunhao123) [reply .changeset/sweet-monkeys-drive.md:1]: @illionillion done

**Review by illionillion — APPROVED** [review]
LGTM
Thanks .

**Comment by hirotomoyamada** [comment]
Thanks!

**Pull request merged**